crdschurch/isgodreal-com
# US18465 Add Imgix to image grid

![Screen Shot 2019-11-12 at 10 37 23 AM](https://user-images.githubusercontent.com/56852436/68685690-7eb01d80-0538-11ea-97fe-1cb559d1dd50.png)

## Task

Replace all `<img />` source attributes with Imigix URLs

[Original Task Link](https://rally1.rallydev.com/#/38108142417d/custom/292180449324?qdp=%2Fdetail%2Fuserstory%2F347613282556)

## Solution

1. Each image has been uploaded to the int environment in Contentful
2. A new `img_grid` value for the required Imgix parameters has been added to `imgix_params` in `_config.yml`
3. All images have been output using the `imgix` filter, `imgix_params.img_grid` and int environment image URL

## Tester Notes

- It will not work on your local machine. **(If there is a way to solve this please let me know)**
- Verify that there is no visual change from [int](https://int-isgodreal-com.netlify.com/) to the [deploy preview](https://deploy-preview-29--int-isgodreal-com.netlify.com/)
- Verify that all image grid images are loading from **crds-media.imgix.net**

## Preview

[Preview Link](https://deploy-preview-29--int-isgodreal-com.netlify.com/)
